.github/workflows/test-vurnabilities-data.yml

```
@@ -0,0 +1,143 @@
+          DB_USER: ${{ secrets.CYPRESS_DB_USER }}
+          DB_PWD: ${{ secrets.CYPRESS_DB_PWD }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u "${{ github.actor }}" --password-stdin
+          chmod +x scripts/trivy_vulnerabilities_data.sh
+          ./scripts/trivy_vulnerabilities_data.sh \
+            "${{ inputs.image_name }}" \
+            "${{ github.event.pull_request.number }}" \
+            "${{ github.event.pull_request.html_url }}" \
+            "${{ github.run_id }}"
+            
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)